config_sample/config.yaml

```diff
@@ -82,3 +82,9 @@ max_chars: 256
 
 # How long a ban will last when no duration is given
 default_ban_duration: 6 hours
+
+# Kicks idlers
+idle_timeout:
+  use_idle_timeout: false
+  kick_mods: false
```

Comment:
What's the purpose of the `kick_mods` setting? I don't see you using it anywhere